packages/apps-config/src/api/spec/index.ts

```diff
@@ -119,6 +119,7 @@ const spec: Record<string, OverrideBundleDefinition> = {
   'cumulus-subsocial-parachain': subsocial,
   'cumulus-test-parachain': testPara,
   datahighway: westlake,
+  'datahighway-harbour': westlake,
```

Comment:
should be harbour right?

Reply:
see the latest changes that were made on 11th June https://github.com/DataHighway-DHX/apps/pull/1/files#diff-d568a276d092ce758fa4d106eaab1ec290619f13479dae43c98bbae34832364fR121